src/utils/email.js

```diff
@@ -89,8 +89,11 @@ function appendUtmParams (url, campaign, content) {
 }
 
 function getEmailCtaHref (emailType, content, subscriberId = null) {
-  const subscriberParamPath = (subscriberId) ? `/?subscriber_id=${subscriberId}` : '/'
-  const url = new URL(subscriberParamPath, SERVER_URL)
+  const dashboardUrl = `${SERVER_URL}/user/breaches`
+  const subscriberParamPath = subscriberId
+    ? `/?subscriber_id=${subscriberId}`
+    : ''
+  const url = new URL(subscriberParamPath, dashboardUrl)
```

Comment:
Would it be possible to use `dashboardUrl.searchParams.set()` here instead?

Reply:
You could, but you'd also get a different URL (so there might be a bug somewhere here).

UUIC, because of the leading `/?` in `/?subscriber_id=${subscriberId}`, it's going to remove the `/user/breaches` part of the path in the code [if `subscriberId` is truthy].
Which means this could would either redirect you to one of the following URLs:
- `{ url: 'https://monitor.firefox.com/?subscriber_id=7' }`
- `{ url: 'https://monitor.firefox.com/user/breaches' }`

```js
const SERVER_URL = "https://monitor.firefox.com";
let subscriberId = "7";

const dashboardUrl = `${SERVER_URL}/user/breaches`;
const subscriberParamPath = subscriberId
  ? `/?subscriber_id=${subscriberId}`
  : '';
const url = new URL(subscriberParamPath, dashboardUrl);
console.log({ url: url.href });
// { url: 'https://monitor.firefox.com/?subscriber_id=7' }
// if `subscriberId` is falsy, you'll get `{ url: 'https://monitor.firefox.com/user/breaches' }`.

const url2 = new URL(dashboardUrl);
url2.searchParams.set("subscriber_id", subscriberId);
console.log({url2: url2.href});
// { url2: 'https://monitor.firefox.com/user/breaches?subscriber_id=7' }
```

Reply:
@pdehaan Thank you for laying out the different scenarios here and providing examples! Stripping `/user/breaches` in case `subscriberId` is truthy is not the behavior I intended to implement.